elements/component.es

```diff
@@ -50,47 +46,27 @@ const Component = Element => // why buble
 
       .map (this.reflect, this)
 
-    // dispatch `idle`
-    // and captured from `EventTarget`
     super.onidle && super.onidle ()
   }
 
-  // This doesn't go here. Perhaps SlotList / Template / TokenList (in that order)
-  parse (template) {
-
-    const
-      fragment = template.content
```

Comment:
@RobertChristopher @brandondees this was the bug. The `.content` attribute is a `DocumentFragment`. so we clone the contents of the template. However when we call `slot.parentNode` below on line 84 it is still referenced to the original imported `<template>`. Therefore the children never get replaced in the clone. (see line 71
`this.innerHTML = template.innerHTML`. The template's `innerHTML` is always going to be the original content because we are working on an in memory `DocumentFragment` content clone that never gets used.

Hope this makes sense. /cc @btakita @mrbernnz @janz93